docs/advanced-features/tabulator.md

```diff
@@ -88,10 +91,29 @@ Console, Athena views, or JDBC connectors. This is especially useful for
 customers who want to access Tabulator from external services, such as Tableau
 and Spotfire.
 
+### Continuing After Errors
+
+> Available since Quilt Platform version 1.58
+
+If Tabulator encounters an error while processing a file, it normally stops and
+returns an error. However, if `continue_on_error` is set to true, it processes
+the remaining files and records any issues in the `$issue` column.
+
+![Tabulator Issue](../imgs/tabulator-issue.png)
+
+If a file is missing a specified column:
+
+- If `nullable` (the default), the column is filled with null, while other
+  columns for all rows remain populated.
+
+- If not nullable, the file is skipped, and a single row of nulls for all
```

Comment:
the value are nulls only if the column is nullable, otherwise they are zeroes for numeric columns and empty strings for string columns.

Reply:
So, let me see if I get this straight:

1. If the column is nullable, the row is always set to null
2. If the columns is NOT nullable, then a) if continue_on_error is set, they are zeroes for numeric columns and empty strings for string columns, b) if not set, it errors out the entire job

Right?

Reply:
right

Reply:
Follow-on at: https://github.com/quiltdata/quilt/pull/4346